webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -92,6 +92,11 @@ def json_dumps(data):
     return jinja2.Markup(json.dumps(data))
 
 
+@library.filter
+def to_json(data):
+    return json.dumps(data)
```

Comment:
I don't like this. It's dangerous. If you do something like this in a jinja template:
```
<script>var DATA = {{ some_view_python_data | to_json }};</script>
```
**and** somewhere inside that `some_view_python_data` there's a `</script>` it'll actually render. E.g. Suppose that you change your name from "Adrian" to `</script><script>alert('xss')</script>`, then you'd get this:
```html
<script>var DATA = {"name": "</script><script>alert('xss')</script>"};</script>
```
and you have an XSS hole.